### ISSUE TITLE
ModuleNotFoundError (it is installed and appears with pip list)

### ISSUE THREAD

**Issue by VictorCallejas** [neptune-ai/neptune-client#354]
Hi,

Running in VSCode JupyterNotebooks with python enviroment 3.8.2
neptune-client 0.4.130

when I try to:
`import neptune`

I get this error:

> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> <ipython-input-2-8b15a7586676> in <module>
> ----> 1 import neptune
>       2 
>       3 import pandas as pd
>       4 import numpy as np
>       5 
> 
> ModuleNotFoundError: No module named 'neptune'

And yes, the library is installed:
`!pip list`

> Package                Version
> ---------------------- -----------
> ...                                 6.0.7
> nbformat                    5.0.8
> **neptune-client         0.4.130
> neptune-notebooks   0.0.16**
> nest-asyncio                1.4.3
> ...
> 

I have tried with different versions but nothing, any idea?

Thank you

**Comment by szymon-kuklewicz** [comment]
Hi,

It would be very helpful if you could provide some reproduction steps from empty python environment to the state that you've described. If possible, using command line only, so we can follow the steps in our environment.

Right now, the reason of such behavior that is most probable, that Jupyter's kernel needs to be restarted after installing new libraries. Try restarting the kernel and check again.

Hope to hear you soon,
Simon

**Comment by VictorCallejas** [comment]
I restarted the kernel multiple times and nothing. I also uninstalled and install different versions and nothing.

I created a new empty environment and it worked. So at the end I deleted my environment and install all dependencies again and it worked.

I do not know what conflict was in my previous environment, but if someone gets this error, **make a pip freeze, delete the env and recreate it again,** for me worked this way.

Anyway thanks for you help Simon and merry Christmas
